### ISSUE TITLE
Is Main Site Down?

### ISSUE THREAD

**Issue by kJamesy** [AMDG2/SimpleHTMLToPDF-API-Wrapper#4]
Hi, I have been using this wrapper in a few of my projects but over the last couple of months, I realise the main site (http://www.simplehtmltopdf.com/) has been down.
Would you by any chance know how to get through to the parties involved?
I will be willing to donate a Digital Ocean box to get it up if it's a hosting issue.

**Comment by AMDG2** [comment]
please contact @Superdrac for more information. I think the Simple HTML to PDF project is ended.

**Comment by Superdrac** [comment]
Yes @kJamesy , the main site is down, sorry. Hosting was too expensive. 

Contact me here if you want the sources. superdrac@gmail.com
